Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -799,256 +771,202 @@ public final class ManifestLoader: ManifestLoaderProtocol {
+            // and validates it.
 
-        // Compute the path to runtime we need to load.
-        let runtimePath = self.toolchain.swiftPMLibrariesLocation.manifestLibraryPath
+            // Compute the path to runtime we need to load.
+            let runtimePath = self.toolchain.swiftPMLibrariesLocation.manifestLibraryPath
 
-        // FIXME: Workaround for the module cache bug that's been haunting Swift CI
-        // <rdar://problem/48443680>
-        let moduleCachePath = try (ProcessEnv.vars["SWIFTPM_MODULECACHE_OVERRIDE"] ?? ProcessEnv.vars["SWIFTPM_TESTS_MODULECACHE"]).flatMap{ try AbsolutePath(validating: $0) }
+            // FIXME: Workaround for the module cache bug that's been haunting Swift CI
+            // <rdar://problem/48443680>
+            let moduleCachePath = try (ProcessEnv.vars["SWIFTPM_MODULECACHE_OVERRIDE"] ?? ProcessEnv.vars["SWIFTPM_TESTS_MODULECACHE"]).flatMap{ try AbsolutePath(validating: $0) }
 
-        var cmd: [String] = []
-        cmd += [self.toolchain.swiftCompilerPathForManifests.pathString]
+            var cmd: [String] = []
+            cmd += [self.toolchain.swiftCompilerPathForManifests.pathString]
 
-        if let vfsOverlayPath {
-            cmd += ["-vfsoverlay", vfsOverlayPath.pathString]
-        }
+            if let vfsOverlayPath {
+                cmd += ["-vfsoverlay", vfsOverlayPath.pathString]
+            }
 
-        // if runtimePath is set to "PackageFrameworks" that means we could be developing SwiftPM in Xcode
-        // which produces a framework for dynamic package products.
-        if runtimePath.extension == "framework" {
-            cmd += [
-                "-F", runtimePath.parentDirectory.pathString,
-                "-framework", "PackageDescription",
-                "-Xlinker", "-rpath", "-Xlinker", runtimePath.parentDirectory.pathString,
-            ]
-        } else {
-            cmd += [
-                "-L", runtimePath.pathString,
-                "-lPackageDescription",
-            ]
-#if !os(Windows)
-            // -rpath argument is not supported on Windows,
-            // so we add runtimePath to PATH when executing the manifest instead
-            cmd += ["-Xlinker", "-rpath", "-Xlinker", runtimePath.pathString]
-#endif
-        }
+            // if runtimePath is set to "PackageFrameworks" that means we could be developing SwiftPM in Xcode
+            // which produces a framework for dynamic package products.
+            if runtimePath.extension == "framework" {
+                cmd += [
+                    "-F", runtimePath.parentDirectory.pathString,
+                    "-framework", "PackageDescription",
+                    "-Xlinker", "-rpath", "-Xlinker", runtimePath.parentDirectory.pathString,
+                ]
+            } else {
+                cmd += [
+                    "-L", runtimePath.pathString,
+                    "-lPackageDescription",
+                ]
+    #if !os(Windows)
```

Comment:
nit: The OS if statements are now indented in a non-standard way

Reply:
Previously the #if was flush left with the file. Now it is one indentation in, but the native level of indentation here is three levels deep. Maybe it is just how GitHub is rendering, but 1 indentation seemed odd because nothing else was at that level